src/Car.js

```diff
@@ -0,0 +1,36 @@
+import { pickNumberInRange, printResult } from './util/missionUtil.js';
+
+class Car {
+  #name;
+  #distance = '';
+  #min = 0;
+  #max = 9;
+
+  constructor(name) {
+    this.#name = name;
+  }
+
+  async getMoveForwardResult() {
+    const RANDOM_VALUE = await pickNumberInRange(this.#min, this.#max);
+    const IS_OVER_4 = RANDOM_VALUE >= 4;
+    if (IS_OVER_4) {
+      this.setDitance('-');
+    }
```

Comment:
- 코드 가독성 측면을 위해, if문 앞뒤에는 줄바꿈이 들어가면 좋을 것 같습니다.
- `4`를 상수로 선언하여 해당 값이 어떤 의미를 담고 있는지 명확히 드러내면 좋습니다. (하단 suggestion 참고)
- 15번째 line에서 변수를 새로 할당하게 되면, 불필요한 메모리할당이 일어나게 됩니다~! 그래서 15, 16번째 라인을 함께 쓰면 좋을 것 같습니다.
- 랜덤값은 가변적인 값이기 때문에, 상수 표현 방식보다 일반 변수 표현 방식이 더 적합할 것 같아요 :)

```suggestion
    if (randomValue >= MIN_SCORE_TO_MOVE) {
      this.setDitance('-');
    }
```